src/be/mcjava/view/CustomerMainMenuOverview.fxml

```diff
@@ -11,17 +11,21 @@
 <FlowPane xmlns="http://javafx.com/javafx"
           xmlns:fx="http://javafx.com/fxml"
           fx:controller="be.mcjava.controller.MenuActionController"
-          fx:id="flowpane"
-          prefHeight="400.0" prefWidth="600.0">
+          fx:id="flowpane">
+          <!--prefHeight="400.0" prefWidth="600.0"-->
     <stylesheets>
         <URL value="@mcjava.css"/>
     </stylesheets>
     <VBox fx:id ="firstmenuvbox">
-        <VBox fx:id="headingvbox" prefHeight="40"/>
+        <VBox fx:id="headingvbox" prefHeight="40">
+            <Label text="McMenus"/>
+        </VBox>
         <GridPane fx:id="maingrid">
 
         </GridPane>
-        <VBox fx:id="productsvbox" prefHeight="40"/>
+        <VBox fx:id="productsvbox" prefHeight="40">
+            <Label text="Products"/>
+        </VBox>
         <HBox>
```

Comment:
I found that you can center these within 1 HBox all the others are redundant.
<HBox alignment = "CENTER" spacing = "175">
...(Labels)
<Hbox/>

Reply:
good tip, these will be replaced by PremadeOrderMenu objects, still implemented your change so i can use it when letting the controller fill this section